### PR TITLE
DE46284 - Langterm fixes

### PR DESCRIPTION
**Pull request by jameswklassen**
## Issues 🐛 
There were a number of issues with the langterms for this repo
- ❌ Arabic date ranges incorrectly translated
- ❌ Danish + Welsh translations not appearing
- ❌ Various invisible/nontraditional characters removed

## Solutions ✅ 
- ✔️ Removed invisible RTL characters from Arabic template strings
- ✔️ Exported `LangCyBehavior` and `LangDaBehavior` in `localize-behaviour`
- ✔️ Regenerated `.js` lang files (hadn't been done in a while)
- ✔️ Proactively removed potentially 'problematic' characters from lang files

### CODE REVIEW

File: behaviors/localize-behavior.js [hunk 1]
```diff
@@ -27,7 +28,8 @@ window.D2L.UpcomingAssessments.LocalizeBehaviorImpl = {
 			value: function() {
 				return {
 					'ar': this.ar,
-					'da-dk': this['da-dk'],
+					'cy': this.cy,
+					'da': this.da,
```
Comment (jameswklassen) [inline behaviors/localize-behavior.js:32]: Renaming these lang codes was a change that happened in folio, so I reflected it here.

File: behaviors/localize-behavior.js [hunk 2]
```diff
@@ -53,6 +55,8 @@ window.D2L.UpcomingAssessments.LocalizeBehavior = [
 	D2L.PolymerBehaviors.LocalizeBehavior,
 	window.D2L.UpcomingAssessments.LocalizeBehaviorImpl,
 	window.D2L.UpcomingAssessments.LangArBehavior,
+	window.D2L.UpcomingAssessments.LangCyBehavior,
+	window.D2L.UpcomingAssessments.LangDaBehavior,
```
Comment (jameswklassen) [inline behaviors/localize-behavior.js:59]: the real reason welsh/danish translations weren't occuring

File: build/langterms/ar.js [hunk 3]
```diff
@@ -15,12 +15,12 @@ window.D2L.UpcomingAssessments.LangArBehavior = {
...
-		"checklistItem": "Checklist Item",
-		"chooseDate": "Choose Date:",
+		"checklistItem": "مادة في قائمة التحقق",
+		"chooseDate": "اختيار التاريخ:",
 		"content": "المحتوى",
-		"currentPeriod": "{startDate}‏ - {endDate}‏",
+		"currentPeriod": "{startDate} - {endDate}",
 		"discussion": "المناقشة",
-		"dueDateLongImminent": "{prefix}: {‏dueDate}",
+		"dueDateLongImminent": "{prefix}: {dueDate}",
```
Comment (jameswklassen) [inline build/langterms/ar.js:23]: Invisible RTL characters were breaking the template strings:

image

image

**Review by marcsoiferman — APPROVED** [review]
LGTM, who would have thought invisible characters would exist 🤷

File: build/langterms/ar.js [hunk 4]
```diff
@@ -15,12 +15,12 @@ window.D2L.UpcomingAssessments.LangArBehavior = {
 		"activityExempted": "معفى",
 		"activityOverdue": "تم تجاوز تاريخ الاستحقاق",
 		"assignment": "الفرض",
-		"checklistItem": "Checklist Item",
-		"chooseDate": "Choose Date:",
+		"checklistItem": "مادة في قائمة التحقق",
+		"chooseDate": "اختيار التاريخ:",
```
Comment (jadasc) [inline build/langterms/ar.js:19]: where did these translations come from?

Comment (jadasc) [reply build/langterms/ar.js:19]: oh wait, this file was regenerated 👍

**Review by jadasc — APPROVED** [review]

**Review by EvanLemieux — APPROVED** [review]

File: build/langterms/ar.js (hunk 4 above)
Comment (jameswklassen) [reply build/langterms/ar.js:19]: yeah, we hadn't regenerated the JS files in a while.... 😓

**Pull request merged**